source/guide/3.2/ui-blueprint-upload.md

```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
 ![The blueprint upload button](/guide/images/ui/ui_upload_blueprint_button.png)
 
 
-Once the blueprint upload dialog appears, the user can select a blueprint TAR file from the filesystem by pressing the `+` button:
+Clicking on it will cause the blueprint upload dialog to appear. <br>
+The user can either type in the path to the blueprint TAR file, or select it from the filesystem by pressing the `+` button:
```

Comment:
Is it only a TAR file? Didn't we add support for other formats?

Also, does the blueprint file have to be named "blueprint.yaml" ?